fbpic/particles/particles.py

```diff
@@ -634,7 +652,7 @@ def push_x( self, dt, x_push=1., y_push=1., z_push=1. ) :
                 self.inv_gamma, self.Ntot,
                 dt, x_push, y_push, z_push )
 
-    def gather( self, grid ) :
+    def gather( self, grid, envelope_grid = None ) :
```

Comment:
I think that the argument `envelope_grid` could be removed. (It probably comes from a previous implementation of the gathering.)